frontend/micro-ui/web/micro-ui-internals/packages/modules/im/src/pages/employee/ComplaintDetails.js

```
@@ -61,37 +61,36 @@ const CloseBtn = (props) => {
 const TLCaption = ({ data, comments }) => {
-  const { t } = useTranslation()
+  const { t } = useTranslation();
   return (
     <div>
       {data?.date && <p>{data?.date}</p>}
       <p>{data?.name}</p>
       <p>{data?.mobileNumber}</p>
       {data?.source && <p>{t("ES_COMMON_FILED_VIA_" + data?.source.toUpperCase())}</p>}
-      {comments?.map( e => 
+      {comments?.map((e) => (
         <div className="TLComments">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)